controller/tokenization/tokenization_service.py

```diff
@@ -25,6 +25,18 @@ def request_tokenize_project(project_id: str, user_id: str) -> None:
     service_requests.post_call_or_raise(url, data)
 
 
+def request_tokenize_calculated_attribute(
+    project_id: str, user_id: str, attribute_name: str
+) -> None:
+    url = f"{BASE_URI}/tokenize_calculated_attribute"
+    data = {
+        "project_id": str(project_id),
+        "user_id": str(user_id),
+        "attribute_name": str(attribute_name),
```

Comment:
is there a specific reason we use the name and not the id?

Reply:
Changed to ID